common/smoke-test/.gitignore

```diff
@@ -1,4 +1,5 @@
-!package-lock.json
+package-lock.json
```

Comment:
I know you didn't add this but is there a reason we need to have a .gitignore file scoped to this directory? In general we try to keep one for the entire repo at the root. Can these settings be merged into that one?

Reply:
Yeah for package-lock that can be inherited. For my addition of `run-manifest.json`, my only thinking in favor of it being scoped is that it's an auto-generated file specific to this directory, so putting it in a top-level .gitignore removes the context of why/where it's being ignored. I can add it to the top level with a comment though?